src/app/infinite-scroller.ts

```diff
@@ -11,9 +11,23 @@ export class InfiniteScroller {
     private pageSize: number,
     private getPage: (page: number) => any[] | Promise<any>,
     private windowViewport: boolean,
-    private bufferSize: number = 50
+    private bufferSize: number = 50,
+    private padding: number = undefined
```

Comment:
When padding isn't defined on an ngx-ui-scroll component, it defaults  to 0.5. This will allow us to have a single settings object.

https://github.com/dhilt/ngx-ui-scroll#settings

```suggestion
    private padding: number = 0.5
```

Reply:
Ooh thanks for pointing this out! I wasn't entirely sure how these settings are being used.